### PR TITLE
chore(*): remove sample access key from SDK documentation

### PR DESCRIPTION
**Pull request by outsinre**
In the AWS SDK, there is sample access key sample `wJalrXUtnFEMI` that may trigger CVE report. This PR tries to remove it.

This is a dangerous operation! Before merge, we must ensure only the access key sample is removed!

FTI-5732 FTI-5732

### CODE REVIEW

**Comment by outsinre** [comment]
```
$ for i in ./delete-me/apis/*.normal.json ; do jdiff "$i" ~/workspace/aws-sdk-js/apis/$(basename $i) ; done
{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}
{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{"operations": {"GetAccessKeyInfo": {"documentation": "<p>Returns the account identifier for the specified access key ID.</p> <p>Access keys consist of two parts: an access key ID (for example, <code>AKIAIOSFODNN7EXAMPLE</code>) and a secret access key (for example, <code>wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY</code>). For more information about access keys, see <a href=\"https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html\">Managing Access Keys for IAM Users</a> in the <i>IAM User Guide</i>.</p> <p>When you pass an access key ID to this operation, it returns the ID of the AWS account to which the keys belong. Access key IDs beginning with <code>AKIA</code> are long-term credentials for an IAM user or the AWS account root user. Access key IDs beginning with <code>ASIA</code> are temporary credentials that are created using STS operations. If the account in the response belongs to you, you can sign in as the root user and review your root user access keys. Then, you can pull a <a href=\"https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_getting-report.html\">credentials report</a> to learn which IAM user owns the keys. To learn who requested the temporary credentials for an <code>ASIA</code> access key, view the STS events in your <a href=\"https://docs.aws.amazon.com/IAM/latest/UserGuide/cloudtrail-integration.html\">CloudTrail logs</a> in the <i>IAM User Guide</i>.</p> <p>This operation does not indicate the state of the access key. The key might be active, inactive, or deleted. Active keys might not have permissions to perform an operation. Providing a deleted access key might return an error that the key doesn't exist.</p>"}}}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}{}
```

image

**Comment by outsinre** [comment]
Please do not merge! We are waiting for customer feedback.

**Comment by windmgc** [comment]
I'm considering whether removing all the document fields is a good idea since we don't actually need those things and that could probably also help us maintain smaller JSON files

File: update_api_files.sh
Comment (Tieske) [inline update_api_files.sh:78]:
```suggestion
  # remove example keys from documentation to prevent security reports from being triggered
  jq 'walk( if (type == "object") and has("documentation") and (.documentation|contains("wJalrXUtnFEMI")) then del(.documentation) else . end )' "$source_file" >| "$TFILE"
```

**Comment by outsinre** [comment]
@windmgc we can do it. But I try to minimize the change for this case.

**Review by windmgc — DISMISSED** [review]

**Review by Tieske — APPROVED** [review]
please add a chaneglog entry before merging

File: README.md
Comment (Tieske) [inline README.md:186]:
```suggestion
  avoids false postive vulnerability report.
  [102](https://github.com/Kong/lua-resty-aws/pull/102)
```

**Review by Tieske — APPROVED** [review]

**Pull request merged**
